source/bdDetect.py

```diff
@@ -581,3 +581,7 @@ def driverSupportsAutoDetection(driver):
 	"VID_10C4&PID_EA60", # SuperBraille 3.2
 })
 
+# NattiqBraille

```

Comment:
Could you please position the block above # superBrl, so the entries will stay in alphabetical order?

Reply:
ok done.